build-images.sh

```diff
@@ -0,0 +1,10 @@
+
+#!/usr/bin/env bash
+# Do not edit by hand; please use build scripts/templates to make changes
+set -eo pipefail
+
+docker context create cimg
+docker buildx create --use cimg
+docker buildx build --platform=linux/amd64,linux/arm64 --file 21.0/Dockerfile -t cimg/openjdk:21 -t cimg/openjdk:21 --push .
```

Comment:
imo i think we should get the flag sorted first rather than changing things manually. the 21.0.0 string crops up in a lot of places, so it's very possible we end up missing it somewhere if we are changing it manually

Reply:
@JalexChen done, merged those changes into `cimg-shared` and rebuilt